src/pages/ServiceDetails/ServiceInfo/ServiceInfoDescription.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import ServiceInfoBadge from './ServiceInfoBadge';
+import ServiceInfoCard from './ServiceInfoCard';
+import { Col, Row } from 'patternfly-react';
+
+class ServiceInfoDescription extends React.Component<any> {
```

Comment:
If there is no option, "any" as type is good, but declaring a Type for props and State would help to jump into a more typescript way.

Reply:
+1 @lucasponce apparently there are props (I see `this.props.name` below) so it would be nice to declare them